package.json

```diff
@@ -68,7 +68,7 @@
     "heroku-prebuild": "npx lerna init",
     "jest": "jest",
     "lint:js": "eslint --ext .js,.jsx .",
-    "lint": "npm run lint:js",
+    "lint": "npm run lint:js -- --fix",
```

Comment:
I'm not sure if we want to default to automatically fixing lints; my thinking is to leave this be so developers can see what errors occur and then decide what action to take. I'm curious what others think

Reply:
I deleted it.